components/Paragraph/index.js

```diff
@@ -11,7 +12,13 @@ const Paragraph = ({ children, size = null, maxWidth, align }) => {
       }`}
       style={{ maxWidth }}
     >
-      <Markdown inline tagNames={["a", "b", "code", "em", "strong"]}>
+      <Markdown
+        inline
+        options={{
+          ...defaultSchema,
+          tagNames: ["a", "b", "code", "del", "em", "strong", "sup"],
```

Comment:
Don't quite understand how we're rendering headings and paragraphs if this is the list of limited tagNames, but it seems to work!

Reply:
This is the subset allowed in `Paragraph` only

Reply:
`Markdown` gets its tags from the `defaultSchema`